changelogs/fragments/6819-redfish-utils-add-links-parameter-for-get_disk_inventory.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - redfish_utils - support ``Links`` parameter for ``GetDiskInventory`` (https://github.com/ansible-collections/community.general/pull/6819).
```

Comment:
parameter or return value?

Reply:
parameter inside return value technically.

Reply:
any changes needed here?

Reply:
Given the direction of things, I think it would be more accurate to state:

> support ``Volumes`` in response to ``GetDiskInventory``

Reply:
Updated as suggested. Thanks!

Reply:
Resolving this conversation. Thanks!